tests/test_multislice.py

```diff
@@ -15,8 +16,10 @@ def test_exit_wave_to_image():
     ).astype(np.float32)
     ones = np.ones((n_pixels, n_pixels))
     exit_wave = sphere.sum(-1)
-    exit_wave_f = fourier.do_fft(exit_wave, dim=2)
-    high_dose = 1e9 * exit_wave.max()
+    exit_wave_torch = torch.tensor(exit_wave).reshape(1, 1, n_pixels, n_pixels)
+    exit_wave_f_torch = transforms.primal_to_fourier_2D(exit_wave_torch)
+    exit_wave_f = exit_wave_f_torch.detach().numpy().reshape(n_pixels, n_pixels)
+    high_dose = 1e6 * exit_wave.max()
```

Comment:
I changed this becuase Luis said 1e9 was too high for his windows comp. The `atol=1e-2` is so that it passes. I tested 10 000 times (stochastic poisson inside) and they all passed, so should be good.